src/flow-bit.c

```diff
@@ -142,14 +187,20 @@ static int FlowBitTest01 (void)
 
     Flow f;
     memset(&f, 0, sizeof(Flow));
+    DetectEngineCtx *de_ctx = NULL;
 
-    FlowBitAdd(&f, 0);
+    de_ctx = DetectEngineCtxInit();
+    FAIL_IF_NULL(de_ctx);
 
+    de_ctx->flags |= DE_QUIET;
+
```

Comment:
you need to make sure `VarNameStoreActivateStaging()` is called. This normally done by `SigGroupBuild` which prepares the detect engine for us after rules have been added to it.

Reply:
```diff
diff --git a/src/flow-bit.c b/src/flow-bit.c
index f4fee1aad..e9d779d87 100644
--- a/src/flow-bit.c
+++ b/src/flow-bit.c
@@ -191,8 +191,8 @@ static int FlowBitTest01 (void)
 
     de_ctx = DetectEngineCtxInit();
     FAIL_IF_NULL(de_ctx);
-
     de_ctx->flags |= DE_QUIET;
+    SigGroupBuild(de_ctx);
 
     FlowBitAdd(&f, 0);
     FlowBit *fb = FlowBitGet(&f,0);
```
gets you
```
Test FlowBitTest01                                                : [3026] 28/3/2020 -- 13:47:28 - (util-classification-config.c:365) <Info> (SCClassConfParseFile) -- Added "34" classification types from the classification file
[3026] 28/3/2020 -- 13:47:28 - (util-reference-config.c:340) <Info> (SCRConfParseFile) -- Added "19" reference types from the reference.config file
[3026] 28/3/2020 -- 13:47:28 - (util-var-name.c:98) <Info> (VariableStoreAddToBloomFilter) -- Inside VariableStoreAddToBloomFilter
[3026] 28/3/2020 -- 13:47:28 - (util-var-name.c:103) <Info> (VariableStoreAddToBloomFilter) -- Adding to bloomfilter, idx: 0
pass
==== TEST RESULTS ====
PASSED: 1
FAILED: 0
======================
```